lib/src/shortest_path.dart

```diff
@@ -134,4 +134,4 @@ Map<T, List<T>> _shortestPaths<T>(
   return distances;
 }
 
-bool _defaultEquals(Object a, Object b) => a == b;
+bool _defaultEquals<T>(T a, T b) => a == b;
```

Comment:
I think this can just be:

```suggestion
bool _defaultEquals(Object? a, Object? b) => a == b;
```

Or not touched if we restrict `T` to `<T extends Object>`

Reply:
Object?